app/models/review.rb

```diff
@@ -0,0 +1,21 @@
+class Review < ActiveRecord::Base
+  belongs_to :video
+  belongs_to :user
+  validates_presence_of :rating
+  validates_presence_of :content
+
+  validate :rating_in_range
```

Comment:
If you want to validate in range, you can validate on inclusion (to some array). Study docs [here](http://guides.rubyonrails.org/active_record_validations.html#inclusion).